src/usr/local/www/services_unbound_host_edit.php

```diff
@@ -225,6 +225,18 @@ function hosts_sort() {
 	));
 }
 
+$section->addInput(new Form_StaticText(
+	'',
+	'<span class="help-block">' .
+	gettext("This page is used to override the usual lookup process for a specific host. A host is defined by its name" .
```

Comment:
Needs a space after `name` so it doesn't run into the next line. It's displaying "nameand" in the output.